src/components/bootstrap-overrides.css

```diff
@@ -1,3 +1,84 @@
 .card {
   border: none;
 }
+.partners-section-header {
+  background: linear-gradient(180deg, rgba(255, 255, 255, 0) 0%, #64c5b4 32.08%),
+    #ffffff;
+}
+.partners-container {
+  background-color: #64c5b4;
+}
+.partners {
+  text-align: left;
+  width: 520px;
+  border: none;
+  background-color: #64c5b4;
```

Comment:
switched the color value out for the one in figma (remind me to show you this/figure out if you have access to it)